src/Grant/IgExchangeToken.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace League\OAuth2\Client\Grant;
+
+class IgExchangeToken extends AbstractGrant
```

Comment:
Let's rename this `InstagramExchangeToken` for clarity.

Reply:
Turns out this naming was consistent with the expectations of `GrantFactory` 😓 Sorry about that! I renamed it back.